### ISSUE TITLE
Remmina only shows black screen without any text after connection to an SSH server

### ISSUE THREAD

**Issue by e-alfred** [FreeRDP/Remmina#1316]
If I connect to an SSH server, I only get a black screen without any text in Remmina. This seems to be a regression in the latest release, it worked on older 1.2.0 releases. This is what I can see on the console:

```
StatusNotifier/Appindicator support: your desktop does support it and libappindicator is compiled in remmina. Good!

(remmina:20712): Gtk-WARNING **: Theme parsing error: gtk.css:101:18: Not using units is deprecated. Assuming 'px'.

(remmina:20712): Gtk-WARNING **: Theme parsing error: gtk.css:101:20: Not using units is deprecated. Assuming 'px'.

(remmina:20712): Gtk-WARNING **: /build/gtk+3.0-3sSotQ/gtk+3.0-3.10.8/./gtk/gtkcontainer.c:1166: child property `position' of container class `GtkHeaderBar' is not writable

(remmina:20712): Gtk-WARNING **: gtk_menu_attach_to_widget(): menu already attached to GtkMenuItem
```

* Client (OS name and version): Kubuntu 14.04.5
* Remmina version (remmina --version): 1.2.0-rcgit-21-0~remmina201710082254.rc3f630d.de61f735~ubuntu14.04.1
* Desktop environment (GNOME, Unity, KDE, ..): KDE
* Connecting to (OS and version): Ubuntu 16.04, Debian 7/8/9
* Connecting via (RDP, VNC, ...): SSH

**Comment by e-alfred** [comment]
I reinstalled rcgit-20 from the PPA and it works. This is definitely a regression in rcgit-21.

**Comment by giox069** [comment]
I can confirm this issue for Ubuntu 14.04. The issue is not applicable to newer Ubuntu versions (i.e: 16.04 or 17.10).
Changing the "Terminal color scheme" in the profile settings does not fix the issue.

**Comment by antenore** [comment]
@e-alfred may I ask you to test the PR #1318 ?

Hopefully it should fix your issue, but I don't have an Ubuntu 14 to test

**Comment by antenore** [comment]
Finally I've identified and fixed the problem.

![1507500584576 remmina-2017-10-17-21 27 54 066275](https://user-images.githubusercontent.com/5975/31690463-0d20b934-b393-11e7-90e0-c591c3b7654b.png)

I need some more time to clean the code and do some regression tests

**Comment by Ra1da35ma** [comment]
hi, please what can I do or run in terminal to fix this on Ubuntu 17.10

**Comment by giox069** [comment]
@Ra1da35ma have you upgraded to Remmina 1.2.0-rcgit.24 ? If not, use our PPA as explained on the [Remmina Wiki](https://github.com/FreeRDP/Remmina/wiki).
